components/script/dom/htmlelement.rs

```diff
@@ -417,12 +418,19 @@ impl HTMLElementMethods<crate::DomTypeHolder> for HTMLElement {
         element.set_click_in_progress(false);
     }
 
-    // https://html.spec.whatwg.org/multipage/#dom-focus
-    fn Focus(&self, can_gc: CanGc) {
+    // <https://html.spec.whatwg.org/multipage/#dom-focus>
+    fn Focus(&self, options: &FocusOptions, can_gc: CanGc) {
```

Comment:
```suggestion
    /// <https://html.spec.whatwg.org/multipage/#dom-focus>
    fn Focus_(&self, options: &FocusOptions, can_gc: CanGc) {
```

We need to support two DOM focus: one without any argument, the other with FocusOptions.

Pls see https://html.spec.whatwg.org/multipage/interaction.html#dom-focus-dev